docs/trex_sandbox_test.md

```diff
@@ -0,0 +1,201 @@
+
+* All resources must be static files (`.js`, `.css`, images, etc.).
+
+* Resources should be placed in the same folder as your extension web page(s), or in a sub-folder of that directory. All references must be relative.
+
+* No inline scripts or styles. Follow the guidelines that all extensions should follow to reduce XSS vulnerabilities, see [Cross-Site Scripting and Extension Security]({{site.baseure}}/docs/trex_xss_guidance.html).
+
+* Sandboxed Extensions cannot connect to sites outside of the sandbox and cannot transfer data. For example, a Sandboxed extension cannot transfer data to be processed by other applications. If you need to connect to outside services or resources, you can create a Network Enabled extension. Network Enabled extensions can't run in the local sandbox environment or in the Tableau Hosting Cloud Service for Sandboxed Extensions.
+
+* **React.js** - If you are creating your Sandboxed Extension using React.js, routing will not work, including hash-based routing (`.../index.html#/configDialog`) in your URL. If you want to use multiple pages in your extension, the alternatives to routing in React.js include using [react-app-rewired](https://www.npmjs.com/package/react-app-rewired){:target="_blank"}{:ref="noopener"} and [react-app-rewire-multiple-entry](https://www.npmjs.com/package/react-app-rewire-multiple-entry){:target="_blank"}{:ref="noopener"}, or [Gatsby](https://www.gatsbyjs.org/){:target="_blank"}{:ref="noopener"}.
+
+* Sandboxed Extensions must be tested in the local sandbox environment. Issues discovered with the extension must be fixed and verified before submitting the Sandboxed Extension package to Tableau.
+
+---
+
+## Sandboxed Extensions development environment
+
+The Extensions API SDK provides a local development environment that replicates the Tableau Hosting Cloud Service for Sandboxed Extensions. You can test your Sandboxed extensions locally with the same sandbox policies before submitting the extension to Tableau for publication. See [Publish Sandboxed Extensions]({{site.baseurl}}/docs/trex_sandbox_publish.html) for information about making your extension available to others in the cloud service for Sandboxed Extensions. The local development environment can't be used to deploy or publish your extension.
+
+---
+
+## Install and start the local sandbox environment
+
+The following instructions assume that you have downloaded or cloned the Extensions API GitHub project.
+
+1. Go to the `extensions-api` folder.
+
+2. If you haven't done so already, install the web server the sandbox test components:
+
+   ```cmd
+   npm install
+   ```
+
+3. Start the sandbox server:
+
+   ```npm
```

Comment:
This looks like a markdown code block language specifier, like ```js or ```ruby -- but I'm not sure these particular ones are accepted in github -- https://help.github.com/en/articles/creating-and-highlighting-code-blocks#syntax-highlighting

I don't see them in the list

Reply:
Good catch. That was a mess up on my part. I was following the "Throw something against the wall and see if it sticks" method, instead of looking up the keywords: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml 

I changed the instructions to use bold. That's the convention we are supposed to use when providing commands in procedures.